src/test/java/org/cactoos/scalar/OrTest.java

```diff
@@ -144,4 +145,17 @@ public void testFuncVarargs() throws Exception {
             new ScalarHasValue<>(false)
         );
     }
+
+    @Test
+    public void testWithMultipleFunctions() throws Exception {
```

Comment:
please provide additional a test for an opposite situation, eg with new ScalarHasValue<>(false)

Reply:
Fixed